python/cudf/cudf/utils/cudautils.py

```diff
@@ -608,6 +608,44 @@ def apply_round(data, decimal):
     return output_dary
 
 
+@cuda.jit
+def gpu_clip_inplace(in_col, lower, upper):
+    i = cuda.grid(1)
+
+    if i < in_col.size:
+        if in_col[i] < lower:
+            in_col[i] = lower
+        elif in_col[i] > upper:
+            in_col[i] = upper
```

Comment:
Instead of a separate kernel for inplace, you can just use the same kernel and pass the `in_col` as both the `in_col` and the `out_col`.

Reply:
Got it!